contracts/MechMarketplace.sol

```diff
@@ -351,8 +350,7 @@ contract MechMarketplace is IErrorsMarketplace {
         uint256 priorityMechServiceId,
         address requesterStakingInstance,
         uint256 requesterServiceId,
-        uint256 responseTimeout,
-        bytes memory paymentData
```

Comment:
No need fir payment data, if anything additional is needed for the future - can be stored under `requestId` in the mech itself.

Reply:
Hm. I'd just leave a default `data` field.